templates/containers/container-build-service.html

```diff
@@ -0,0 +1,397 @@
+                                alt="",
+                                width="852",
+                                height="1278",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"claas": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Security and compliance</h3>
+            </div>
+            <div class="p-equal-height-row__item">
+              <p>
+                Meet industry standards like CIS benchmarks and FIPS compliance with signed, trusted images.
+              </p>
+            </div>
+          </div>
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/1220fee3-Enterprise-wide requirements.png",
+                                alt="",
+                                width="852",
+                                height="1278",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"claas": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Enterprise-wide requirements</h3>
+            </div>
+            <div class="p-equal-height-row__item">
+              <p>
+                Ensures every image includes packages and libraries that simplifies management and troubleshooting, which may fulfill enterprise -wide requirements.
+              </p>
+            </div>
+          </div>
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/882080ce-Application-specific customization.png",
+                                alt="",
+                                width="852",
+                                height="1278",
+                                hi_def=True,
+                                loading="lazy") | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Application-specific customization:</h3>
```

Comment:
```suggestion
              <h3 class="p-heading--5">Application-specific customization</h3>
```
To match the other headings, I have updated the copydoc

Reply:
Thank you!